release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+# Overview of jBallerina 1.2.0
+
+JBallerina 1.2.0 is the first major Q1 release of 2020 and it includes a new set of features and significant improvements to the compiler, runtime, standard library and developer tooling. It is based on Stable Language Spec version 2020R1.
+
+# Highlights
+
+- Redesigned XML, Stream and Query support
+- Based on a stable language specification: 2020R1
+- Improved Compilation time.
+- Includes a number of important bug fixes
```

Comment:
Shall we mention bug fixes at the end of this list, after all the features?

Reply:
Done